### PR TITLE
Remove undefined aria-label tests.

### PR DESCRIPTION
**Pull request by jcsteh**
@nmlapre discovered this in his triaging and investigation of Gecko WPT failures.

Many ARIA attributes do support a value of "undefined". However, [aria-label](https://w3c.github.io/aria/#aria-label) expects a string value, which is [specified](https://w3c.github.io/aria/#valuetype_string) as an "Unconstrained value type." Thus, "undefined" is not a special value for aria-label, and these tests are incorrect. [All browsers currently fail these tests](https://wpt.fyi/results/accname/name/comp_label.html?label=master&label=experimental&aligned&q=status%3Afail).

I looked for other references in the ARIA spec that suggest that aria-label should support "undefined", but I found none. The [true/false/undefined](https://w3c.github.io/aria/#valuetype_true-false-undefined) and [tristate](https://w3c.github.io/aria/#valuetype_tristate) explicitly specify that they support undefined, but no other types do.

### CODE REVIEW

**Review by MelSumner — APPROVED** [review]
Re-read the relevant portions of spec and agree with this assessment.

**Review by zcorpan — APPROVED** [review]
LGTM, but this should get signoff in https://github.com/web-platform-tests/interop/issues/648 before merging.

**Review by cookiecrook — APPROVED** [review]

**Pull request merged**
